eng/azure-sdk-deps.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFramework>netstandard2.0</TargetFramework>
+    <EnableDefaultItems>false</EnableDefaultItems>
+    <PublishDepsFilePath>$(IntermediateOutputPath)\azure-sdk-deps.json</PublishDepsFilePath>
+    <SdkRootPath>..\sdk</SdkRootPath>
+  </PropertyGroup>
+
+  <ItemGroup>
+    <ExcludeProjects Include="$(SdkRootPath)\**\*Test*.csproj" />
+    <IncludeProjects Include="$(SdkRootPath)\*\Azure.*\src\Azure.*.csproj" />
```

Comment:
Did you intend to include the MgmtClient libraries?

Reply:
Yes I did intend to include the track2 mgmt libraries.